rust-version

```diff
@@ -0,0 +1 @@
+1b61d43bdbf875183b1f436302d62ff93f9a6bba
```

Comment:
ideally CI would use this to determine the Rust version to test, that'd made it stable under rustc repo changes... but that's a future TODO I think :)

Reply:
Specifically we have a nightly CI job that [starts with](https://github.com/rust-lang/miri/blob/645b974b22d9a22dbe393af1729b51526cdab7a1/.github/workflows/setup/action.yml#L48):
```
git ls-remote https://github.com/rust-lang/rust/ HEAD | cut -f 1 > rust-version
```
So we test against the latest rustc all the time, but even if rustc breaks, that doesn't affect unrelated PRs. If CI fails in a PR, that's *always* the fault of the PR because everything is pinned. IMO that is a quite important property to have.

Reply:
That is nice, yeah. I'll take a look at it once we have rustc-josh-sync for stdarch up & running.